ASP.NET Core/Readme.md

```diff
@@ -0,0 +1,54 @@
+# ASP.NET Core
+
+## Installation
+
+Download the example and use Visual Studio 2019 to open the project. 
+
+## Client-side resources and bundling
+
+This project uses [NPM](http://npmjs.com/) and [Gulp.js](https://gulpjs.com/) to install client-side libraries. All required scripts and CSS files will be automatically installed and packaged into the resulted bundle during the project first build.
+
+The resulted bundles will be located in the `wwwroot` folder:
+* `css/vendor.css` - a file with all CSS styles.
+* `css/icons` and `css/fonts` - folders that contains fonts and icons for DevExtreme themes.
+* `js/vendor.js` - a file that contains all scripts.
+
+The default bundle includes jQuery, Bootstrap, and DevExtreme.
+
+### Add more 3rd-party libraries for additional features/components 
+
+The main logic is located in the the `gulpfile.js` file at the root application level. The file contains two tasks for copying resources from NPM to the project:
```

Comment:
What do you think about this variant?
The file contains two default tasks:
* `add-resouces` -  this task adds fonts\icons, "JS" files located in the `scripts` array and CSS located in the "styles" array to the resulting bundle and coping it to wwwroot.
* `clean` - this task removes all previously created files (`vendor.js` and `vendor.css`) and folders (`icons` and `fonts`).

The main idea is to remove the incorrect part from the first sentence and add some additional info to the first task description.

Reply:
done